aliases.yaml

```diff
@@ -0,0 +1,62 @@
+# Aliases from custom name to an array of collection names
```

Comment:
Why an array?

Reply:
I imagine it could be useful an alias to expand out into multiple collections in a query